lightweight/common/eventdata.cpp

```diff
@@ -0,0 +1,23 @@
+#include "eventdata.h"
+
+QDBusArgument &operator<<(QDBusArgument &argument, const EventData &eventData)
+{
+    argument.beginStructure();
+    argument << eventData.displayLabel << eventData.description << eventData.startTime
+             << eventData.endTime << eventData.recurrenceId << eventData.allDay
```

Comment:
Recurrence id in the middle of time information seems illogical. Could perhaps arrange the content even otherwise to have more grouping, e.g. calendar id, event id, recur id, times, event details in descending importance.